### DISCUSSION TITLE
New Living Translation (NLT) request

### DISCUSSION THREAD

**Discussion by Luke2791** [tim-hub/obsidian-bible-reference#202] [Ideas]
Hi there! I love this plug in - it's fantastic! Are there any plans for adding the New Living Translation (NLT) in a future release? The NLT is easy to read and great for non-native English speakers. This addition would be a blessing. I currently use ESV, but frequently find myself googling the NLT.

**Comment by tim-hub** [comment]
Added in 2.5.6 version :)

**Reply by Luke2791** [reply]
Amazing!!! Wow - thank you so much! Just bought you a $5 "coffee" to say thanks! Appreciate the support :). I do think it's a great add as well - love the NLT. Blessings to you!

**Reply by tim-hub** [reply]
Thanks :) 
God bless
